src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesWorkspaceBrowser.java

```diff
@@ -21,20 +21,25 @@
 
     @Override
     public FilePath getWorkspace(Job job) {
-        LOGGER.info("Nodes went offline. Hence fetching it through master");
-        if(job instanceof AbstractProject) {
-            String assignedLabel = ((AbstractProject) job).getAssignedLabelString();
-            KubernetesCloud kubernetesCloud = Hudson.getInstance().clouds.get(KubernetesCloud.class);
-            if(kubernetesCloud != null) {
-                List<PodTemplate> podTemplates = kubernetesCloud.getTemplates();
-                for(PodTemplate podTemplate : podTemplates) {
-                    if(ObjectUtils.equals(podTemplate.getLabel(), assignedLabel)) {
-                        String workspacePath = Jenkins.getInstance().getWorkspaceFor((TopLevelItem) job).toString();
-                        LOGGER.info("Workspace Path: " + workspacePath);
-                        File workspace = new File(workspacePath);
-                        LOGGER.info("Workspace exists ? " + workspace.exists());
-                        if (workspace.exists()) {
-                            return new FilePath(workspace);
+        if(job != null) {
```

Comment:
You don't need to check if job is null. A null job will never go through `job instanceof AbstractProject`.